controller/search.js

```diff
@@ -42,7 +47,7 @@ function setup( backend, query ){
         res.data = docs;
         res.meta = meta;
       }
-
+      logger.debug('[ES response]', JSON.stringify(docs));
```

Comment:
These came in accidentally through a bad rebase. They will go away before merging